README.md

```diff
@@ -6,27 +6,23 @@
     Symfony UX: a new JavaScript ecosystem for Symfony
 </h3>
 
-Since its creation, JavaScript has always been focused on creating innovative User
-Experiences. It lets developers create the **most intuitive and usable** User
-Experience possible for a specific problem.
-
-In practice however, building great User Experiences with JavaScript is difficult.
-It takes time to choose reliable packages, to configure them, to integrate them
-in your pages, and to make your front-end code interact with the rest of your
-infrastructure.
-
-This problem is not new: it looks a whole lot like the state of Symfony in PHP
-before Symfony Flex. We need a Symfony Flex equivalent for JavaScript: a tool to
-build amazing User Experiences as quickly as we can now setup
-[an HTTP client](https://symfony.com/doc/current/http_client.html),
-[a Mailer](https://symfony.com/doc/current/mailer.html) or
-[an administration panel](https://symfony.com/doc/current/bundles/EasyAdminBundle/index.html).
+Symfony UX is an initiative and set of libraries for seamlessly
+integrating JavaScript tools into your application. For example,
+want to render a chart with [Chart.js](https://www.chartjs.org/)? Use
```

Comment:
I'm not sure about this sentence, it feels a bit too familiar for the official README?

Reply:
I hear what you're saying... but the alternative is to make it more boring sounding... which I don't love. This quickly delivers the benefit of UX. But, I'm open to suggestions.